### ISSUE TITLE
Error in function Interplation_G

### ISSUE THREAD

**Issue by Yannnnnnnn** [carpedm20/BEGAN-tensorflow#27]
hello, i think there is a problem in generator interpolation, in lines 237.
--> self.z_r_loss = tf.reduce_mean(tf.abs(self.x - G_z_r))  
self.x is not normed to[-1,1], compute loss will cause an error
--> x = norm_img(self.x)
--> self.z_r_loss = tf.reduce_mean(tf.abs(x - G_z_r))
maybe it's the right form.
i found params "train_epoch=0" in function "interpolate_G", i think you may have tried to map a real image and it's mirror to noise and interplate to generate new images with different poses but failed, after fixed the problem, you can try to interpolate again.
